### PR TITLE
Add snapshot testing for builds

### PR DESCRIPTION
**Pull request by chee**
This adds a copy of `o-table@7`, `o-forms@7` and `o-colors@4`

it builds them with `occ` and checks they look the same as they did the last time they got built.

i chose `o-table` and `o-forms` because they have a healthy amount of javascript and `o-colors` because it doesn't have any

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +31,9 @@
 	"scripts": {
 		"watch": "npm run-script build -s -- -w",
 		"start": "node -r esm $npm_package_main",
-		"test": "mocha -r esm --recursive test",
+		"run-tests": "mocha -r esm test",
+		"clean-test-components": "echo 'cleaning test/components'; git clean -df -- test/components; git checkout -- test/components/",
+		"test": "npm run-script --silent clean-test-components; npm run-script --silent run-tests; npm run-script --slient clean-test-components",
```
Comment (JakeChampion) [inline package.json]:
```suggestion
		"test": "npm run-script --silent clean-test-components; npm run-script --silent run-tests; npm run-script --silent clean-test-components",
```

File: test/npm.js [hunk 2]
```diff
@@ -108,13 +108,11 @@ describe("createDependencyVersion", () => {
...
 
-	it("throws otherwise", async () => {
+	it("throws otherwise", () => {
 		let nonSemver = "$0.$0.0-0.0.0-0.0.0-yeet$17"
-		let version = `blablahblah#${nonSemver}`
-		expect(npm.createDependencyVersion(["yeet", version])).to.eventually.throw()
-		expect(
-			npm.createDependencyVersion(["yeet", nonSemver])
-		).to.eventually.throw()
+		return npm.createDependencyVersion(["yeet", nonSemver]).catch(error => {
```
Comment (JakeChampion) [inline test/npm.js]: What would happen if this resolved? Would it fail the test?

Comment (chee) [reply test/npm.js]: oof no it wouldn't.

Comment (chee) [reply test/npm.js]: the new one would !

Comment (chee) [reply test/npm.js]: wait, i haven't pushed it yet

Comment (chee) [reply test/npm.js]: ...

Comment (chee) [reply test/npm.js]: the new one would !

**Comment by chee** [comment]
and, in it goes!

**Pull request merged**